rocky/tools/templatetags/ooi_extra.py

```diff
@@ -81,3 +81,8 @@ def index(indexable, i):
 @register.filter
 def pretty_json(obj: dict):
     return json.dumps(obj, default=str, indent=4)
+
+
+@register.filter
+def human_readable(reference_string: str):
+    return Reference.from_str(reference_string).human_readable
```

Comment:
Is building a reference from str really needed (and not overly expensive) when we already have the referenced object in scope when rendering the form?

Reply:
I love the idea of a helper function, but I'd rather have a that function just try and return the OOI classes method if thats possible instead of recreating the class from the ref.

Reply:
Good point, its prefered that the creator of the report gives the whole object as context but sometimes its not as easy (for example when using the PK as key in a dict. Its nice to have this kind of helper in that case. Fortunately, creating a Reference from string is basically a fancy string split and shouldn't be too slow. But I agree with you, it should usually be avoided! 

<img width="534" alt="image" src="https://github.com/minvws/nl-kat-coordination/assets/43830693/194011a6-f226-4e93-8f82-bc202758ae74">